### ISSUE TITLE
Kinect V2 integration

### ISSUE THREAD

**Issue by aiegoo** [aiegoo/kinect#1]
### tracked jointIDs
![image](https://user-images.githubusercontent.com/42961200/197434237-e0b0ccbe-6576-4e81-a7da-3d79995c118d.png)

### setup
For the KinectV2, check [here](https://github.com/aiegoo/archery/issues/9) for more details.

[sample_repo](https://github.com/aiegoo/Azure-Kinect-Samples/tree/master/body-tracking-samples/sample_unity_bodytracking) for kinect dk (azure)

[kinectV2sample](https://github.com/aiegoo/kinect-gesture)

[![kinectV2gesture](https://img.youtube.com/vi/r_32XIz8UbI/0.jpg)](https://youtu.be/r_32XIz8UbI)

> for more details check [here](https://github.com/aiegoo/archery/issues/9)

https://user-images.githubusercontent.com/42961200/194526788-dedf24ea-6477-491b-acaf-46c20ca9a0df.mp4

https://user-images.githubusercontent.com/42961200/194528618-6f85f3d0-6511-457d-af90-97e2f2f5c279.mp4

**Comment by aiegoo** [comment]
## reference for archery mechanism and visual effects (HUD)

[![farcry3](https://img.youtube.com/vi/ZFz4fWiTLhA/0.jpg)](https://www.youtube.com/watch?v=ZFz4fWiTLhA)

[![archery_vr](https://img.youtube.com/vi/jRBeP4_qJ9w/0.jpg)](https://youtu.be/jRBeP4_qJ9w)

**Comment by aiegoo** [comment]
## Bow movement and animation

## Target collision 

## Target movement 

## Archer behavior


https://user-images.githubusercontent.com/42961200/197424303-d27f3e42-8dc1-43b0-99a5-1d70ff90466b.mp4

**Comment by aiegoo** [comment]
### porting to xbox game console
[ms_doc](https://docs.microsoft.com/en-us/windows/uwp/xbox-apps/)

### connecting kinect to a Windows usb


![Image](https://user-images.githubusercontent.com/42961200/188258607-6a425d1e-e212-4e13-a675-41c89fa26a10.png)



![Image](https://user-images.githubusercontent.com/42961200/188258697-a64f20ca-cc08-4f86-958d-6a21a62e386e.png)

### download links
[sdk](https://www.microsoft.com/en-us/download/details.aspx?id=40278)

[runtime](https://www.microsoft.com/en-us/download/details.aspx?id=40277)

[developer_toolkit](https://www.microsoft.com/en-us/download/details.aspx?id=40276)

[동서대학교](https://www.youtube.com/watch?v=m5uTH3S9P9g)



I was confused about developing a game using Unity for Xbox as part of Xbox Live Creators Program so I did a bit of investigation and I think I have an answer for it.

There are two ways one can develop for a game for Xbox using Unity: [Independent Developers Program](https://www.xbox.com/en-CA/developers/id) (aka ID@Xbox) and [Xbox Live Creators Program](https://www.xbox.com/en-US/developers/creators-program/). The enrollment process for these two programs are very different (with ID@Xbox being very difficult) as well as the features you can add to your game on each of them. I should also mention that none of these programs are free and you have to pay a fee for the enrollment.

The Xbox Live Creators Program is a newer program than the ID@Xbox and - for maybe because of that  - some aspects of its documentation on Microsoft's website are not very clear; Also it is very strange that even [Unity's documentation](https://unity3d.com/partners/microsoft/creators-program#:~:text=The%20Creators%20Collection%20is%20a,Store%20on%20Windows%2010%20PCs.) for this program is not clear and just repeating Microsoft's wording and even this current page - which is basically targeting ID@Xbox - causes more confusion for those who want to use Creators Program.

The source of the confusion is that Xbox Live Creators Program can only be used to create UWP applications/games which can run on PC as well as Xbox. What this means is that a game developer who is part of the Creators Program does not need to have access to Xbox specific SDKs - from Microsoft, Unity or any other company - to make their UWP game work on Xbox; In other words in Creators Program there is no need to contact a Microsoft Account Manager - if there is any for this program - or a or Microsoft representative for obtaining licenses to show 3rd party companies like Unity you are a Xbox developer.

So in short for developing games for Xbox in Xbox Live Creators Program one should focus on making their game work as UWP application and that's it; The [UWP on Xbox One](https://docs.microsoft.com/en-us/windows/uwp/xbox-apps) and [Bringing Unity games to UWP on Xbox](https://docs.microsoft.com/en-us/windows/uwp/xbox-apps/development-lanes-unity) articles provides more context here.

I hope Unity and Microsoft could update their pages regarding Creators Program to reflect this as soon as possible to avoid confusions for other developers.
